### PR TITLE
Add Activity Method Calling Status

### PR DESCRIPTION
**Pull request by michaelwehrley**
Add Activity Method Calling Status

* Create delegate method ```action```.
* Write test for ```action```.

[Finishes: #88917838]

### CODE REVIEW

**Comment by ericcf** [comment]
@michaelwehrley looks like a RuboCop error.

**Comment by michaelwehrley** [comment]
Yep, aware and fixing this morning :-)

File: spec/models/activites_spec.rb [hunk 1]
```diff
@@ -19,4 +19,14 @@
     expect(planned_activity1.end_time).to be_a_kind_of(Time)
     expect(planned_activity1.to_serialized[:endTime]).to eq(planned_activity1.end_time.strftime("%b. %e, %Y at %l:%M%p"))
   end
+
+  describe "#activity" do
```
Comment (ericcf) [inline spec/models/activites_spec.rb]: I think you mean `#action` here? Also, my inclination is to delete this spec entirely. I don't know that there's a good justification for testing such a trivial method.

Comment (michaelwehrley) [inline spec/models/activites_spec.rb]: That is a good point, but I would like to keep the method just in case we change ```status_label``` in the future.

Comment (michaelwehrley) [inline spec/models/activites_spec.rb]: @ericcf Nice catch; I made the update to #action.  Thanks!

Comment (ericcf) [inline spec/models/activites_spec.rb]: But you're mocking `status_label` here, so this will continue to pass even if you change the real method...

Comment (michaelwehrley) [inline spec/models/activites_spec.rb]: What if we change the method ```action``` in the host app to call ```happiness``` instead?  Won't it will break?

Comment (ericcf) [inline spec/models/activites_spec.rb]: Yes, and that brings up another point: if your test breaks when the **implementation** of a method changes, the test is too brittle.

Comment (michaelwehrley) [inline spec/models/activites_spec.rb]: Yep, you sold me :smile:

Comment (michaelwehrley) [inline spec/models/activites_spec.rb]: That is an excellent point! Thanks!

**Pull request merged**